alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
+
+COPY etc /etc
+
+RUN rm -r /etc/sv/rails /etc/service/rails \
+  && /root/setup-directories.sh root r /etc/service/postgres /etc/service/buildkite \
+  && /root/setup-directories.sh postgres rw /var/db/postgresql \
+  && ln -s -- /etc/sv/postgres /etc/service/postgres/run \
+  && ln -s -- /etc/sv/buildkite /etc/service/buildkite/run \
+  && echo '@testing-extra https://s3.amazonaws.com/alpine-packages/testing-extra' >> /etc/apk/repositories \
```

Comment:
Should we use a different name for this? It seems like we're not really testing anymore.